pkg/cmd/server/origin/master_config.go

```diff
@@ -790,6 +790,11 @@ func (c *MasterConfig) KubeClient() *kclient.Client {
 	return c.PrivilegedLoopbackKubernetesClient
 }
 
+// OpenShiftClient returns the openshift client object
+func (c *MasterConfig) OpenShiftClient() *osclient.Client {
```

Comment:
also maybe highlight that this will return the privileged client? btw. why we need this function if it is only returning a field that is already public?